flask_pydantic_spec/config.py

```diff
@@ -18,10 +15,9 @@ class Config:
     def __init__(self, **kwargs: Dict[str, Any]) -> None:
         self.PATH: str = "apidoc"
         self.FILENAME: str = "openapi.json"
-        self.OPENAPI_VERSION: str = "3.0.3"
+        self.OPENAPI_VERSION: str = "3.1.0"
```

Comment:
I tried to run this locally with a forked repo of this change but this results in an error:

![image](https://github.com/user-attachments/assets/810b0324-105c-4d2a-b8d2-f59ef8f8a529)

Reply:
Reverting it to `self.OPENAPI_VERSION: str = "3.0.3"` resolves the issue for me.